lib/api/objectGet.js

```diff
@@ -187,12 +193,31 @@ function objectGet(authInfo, request, returnTagCount, log, callback) {
                 dataLocator = setPartRanges(dataLocator, byteRange);
             }
         }
+        if (locationType && externalBackends[locationType]) {
```

Comment:
instead of having the if clause here, just call the backend head and if the backend has head, it will return the needed response, otherwise it should be a noop and just return

Reply:
updated